File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int> mpp;
        for(auto el : nums) 
            mpp[el]++;
        vector<pair<int,int>> container;
        for(auto [num,freq]:mpp)
                container.push_back({freq,num});
        sort(container.begin(),container.end(),greater<>());        
        vector<int> ans;
        for(int i= 0;i<k;i++)
            ans.push_back(container[i].second);
            return ans;        
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> mpp = new HashMap<>();
        int[] ans = new int[2];
        for(int i = 0;i<nums.length;i++){
             if(mpp.containsKey(target-nums[i])){
                   ans[0] = mpp.get(target-nums[i]);
                   ans[1] = i;
                   break;
             }
             mpp.put(nums[i],i);
        }
        return ans;
    }
}
